### PR TITLE
ocr-numbers: add exercise to Delphi track

### PR DESCRIPTION
**Pull request by filiptoskovic**
Added ocr-numbers exercise

### CODE REVIEW

**Comment by filiptoskovic** [comment]
example could be better. I was focused on tests

**Comment by rpottsoh** [comment]
@filiptoskovic you have done an excellent job!  I have made a minor suggestion regarding the example solution.  Other than that I like what you have done just fine.  Very much appreciated, top notch.

I'll look into making the necessary changes to the configuration file.  Once I have that file in order just let me know when you are done and I will merge this.

File: exercises/ocr-numbers/uOcrNumbersExample.pas [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+type
+  TOcrNumbers = class
+  private
+    type
+      TSegment = TList<string>;
+    class procedure CheckFormat(AInp : TArray<string>);
+    class function RecogniseNumber(AInp : TArray<string>) : string;
+    class function RecogniseDigit(ADigit : TArray<string>): char;
+    class function SliceToSegments(AInp : TArray<string>) : TObjectList<TSegment>;
```
[the remark this answers is not in the document]
Comment (rpottsoh) [reply exercises/ocr-numbers/uOcrNumbersExample.pas]: If you don't want to make anymore changes to the example solution that is completely fine.  It is not critical that it be the greatest possible solution to the exercise.  It is only important that it is demonstration of a solution that passes all the tests; which it does.

**Comment by filiptoskovic** [comment]
I have refactored example a little, It had large toxicity, over 3.  So, I have noticed that there are no test that check is last line contains spaces only. I know that exercise stated that 4th line is always blank, but exercise stated that there are 4 lines, but there are a check for row numbers. It's a little bit peculiar, isn't it?

**Pull request closed without merging**